lib/trento/application/integration/checks/adapter/wanda/wanda.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Trento.Integration.Checks.Wanda do
+  @moduledoc """
+  Wanda integration adapter
+  """
+
+  @behaviour Trento.Integration.Checks.Gen
+
+  alias Trento.Messaging
+  alias Trento.Messaging.Mapper
+
+  alias Trento.Integration.Checks.{
+    FlatCatalogDto,
+    FlatCheckDto
+  }
+
+  @impl true
+  def request_execution(execution_id, cluster_id, provider, hosts_settings, selected_checks) do
```

Comment:
I am wondering whether we should talk about `targets` rather than `hosts_settings` :thinking: 

Yet, I understand this is comes from what we currently have and it might be subject to change as we progress, so not a big deal.

Reply:
Exactly, the signature comes from the behaviour. I will rename it to the new nomenclature